File: server/app/tables/bookshelf.py
import sqlalchemy

# Database table definitions.
metadata = sqlalchemy.MetaData()

bookshelfs = sqlalchemy.Table(
    "bookshelf",
    metadata,
    sqlalchemy.Column("id", sqlalchemy.Integer, primary_key=True),
    sqlalchemy.Column("novel_id", sqlalchemy.Integer),
    sqlalchemy.Column("user_id", sqlalchemy.Integer),
    sqlalchemy.Column("created_at", sqlalchemy.DateTime),
    sqlalchemy.Column("updated_at", sqlalchemy.DateTime)
)

File: meteor-reading-server/server/models/bookshelf.js
const Sequelize = require('sequelize');
const db = require('./db');

const { STRING, INTEGER } = Sequelize.DataTypes;

module.exports = db.define(
    'bookshelf',
    {
        novel_id: {
            type: STRING,
            required: true,
        },
        user_id: {
            type: INTEGER,
            required: true,
        },
    },
    {
        timestamps: true,
    }
);
